clang/lib/Sema/SemaExpr.cpp

```diff
@@ -19706,22 +19706,29 @@ static void buildLambdaCaptureFixit(Sema &Sema, LambdaScopeInfo *LSI,
   }
 }
 
+static DeclContext *ignoreReuquiresBodyDecl(DeclContext *DC) {
```

Comment:
```suggestion
static DeclContext *ignoreRequiresBodyDecl(DeclContext *DC) {
```

Reply:
Name here doesn't match what you're doing here.  Perhaps something closer to `getParentIgnoringRequiresBodyDecl`?  Not a great name either, but better I think?

Reply:
I think inline it will be better since we only call it once.